homeassistant/components/plex/server.py

```diff
@@ -121,8 +130,22 @@ def update_platforms(self):
         _LOGGER.debug("Updating devices")
 
         available_clients = {}
+        ignored_clients = set()
         new_clients = set()
 
+        monitored_users = self.accounts
+        known_accounts = set(self.options[MP_DOMAIN].get(CONF_MONITORED_USERS, []))
```

Comment:
known_accounts or allowed/enabled/monitored_accounts?

Reply:
- `monitored_accounts`: Users we want to track. Set to all at first as a default, but possibly pruned down.
- `known_accounts`: The list of accounts that we've seen in the past. Used when new users are added to the Plex server, like when a new share is granted. Need to know if we've seen the account before in order to use the `CONF_IGNORE_NEW_SHARED_USERS` option properly.